### ISSUE TITLE
ignoreAtRules - missing or ambiguous examples

### ISSUE THREAD

**Issue by Nirazul** [stylelint/stylelint#1593]
There are some documentation problems concerning `ignoreAtRules`:

I haven't looked through all rules, but it seems that at least for [at-rule-empty-line-before](http://stylelint.io/user-guide/rules/at-rule-empty-line-before/), the option is available, but not documented.

Another question occured to me when looking at pages with the option documented, for example [block-closing-brace-newline-after](http://stylelint.io/user-guide/rules/block-closing-brace-newline-after/):

As there is no obvious connected example config, the given examples might be valid either way:

> Given:

```js
["if", "else"]`
```

The following patterns are *not* considered warnings:

```css
@if ... {
  color: pink;
} @else if ... {
  color: red;
} @else {
  color: blue;
}
```

...this is also valid if the primary option is: `"always-single-line"`. In other rules, the corresponding config is written near a `Given:` block.
